src/app/rundown-view/services/action-trigger-producer-key-binding.service.ts

```diff
@@ -89,14 +91,35 @@ export class ActionTriggerProducerKeyBindingService implements KeyBindingService
     this.actionTriggersWithAction.clear()
 
     this.actionTriggers.forEach(actionTrigger => {
-      const actionForTrigger: Tv2Action | undefined = this.actions.find(action => action.id === actionTrigger.actionId)
-      if (!actionForTrigger) {
+      let action: Tv2Action | undefined = this.actions.find(action => action.id === actionTrigger.actionId)
+      if (!action) {
         return
       }
-      this.actionTriggersWithAction.set(actionTrigger, actionForTrigger)
+      if (action.type === PlaceholderActionType.CONTENT) {
+        action = this.findActionForContentPlaceholderAction(action as Tv2ContentPlaceholderAction, actionTrigger)
+      }
+      if (!action) {
+        return
+      }
+      this.actionTriggersWithAction.set(actionTrigger, action)
     })
   }
 
+  private findActionForContentPlaceholderAction(placeholderAction: Tv2ContentPlaceholderAction, actionTrigger: ActionTrigger<KeyboardTriggerData>): Tv2Action | undefined {
+    const onAirSegment: Segment | undefined = this.rundown?.segments.find(segment => segment.isOnAir)
+    if (!onAirSegment) {
+      return
+    }
+    const actionsForOnAirSegment: Tv2Action[] = this.actions
+      .filter(action => Math.floor(action.rank) === onAirSegment.rank && placeholderAction.metadata.allowedContentTypes.includes(action.metadata.contentType))
+      .sort((a, b) => a.rank - b.rank)
+    const nthToSelect: number = actionTrigger.data.actionArguments as number // We know this is a number because of Tv2ContentPlaceholderAction.
```

Comment:
The action argument name is `indexToSelect`. It would make sense to use that name here as well.

Reply:
I went with `nthToSelect` because index is useally zero-indexed, so it was a way to beware that this wasn't zero-indexed. I can change it to `indexToSelect` if preferred.

Reply:
Updated

Reply:
I missed the point of it being a 1-based index. I don't think that _nth_ exhibit that differently than _index_. So the added comment is fine enough for me :)

Reply:
Too late. It's been changed to `indexToSelect` ;)